modules/nw-proxy-configure-object.adoc

```diff
@@ -102,23 +96,16 @@ spec:
 ----
 +
 --
-<1> A proxy URL to use for creating HTTP connections outside the cluster. The
-URL scheme must be `http`.
-<2> A proxy URL to use for creating HTTPS connections outside the cluster.
-<3> A comma-separated list of destination domain names, domains, IP addresses or
-other network CIDRs to exclude proxying.
+<1> A proxy URL to use for creating HTTP connections outside the cluster. The URL scheme must be `http`.
+<2> A proxy URL to use for creating HTTPS connections outside the cluster. The URL scheme must be `https`.
```

Comment:
The description of the proxy URL for HTTPS connections is not correct, proxy for https connections may be either http or https. We use http proxies for https connections in testing quite frequently and it works fine. If the user is using an proxy that listens for https connections from the cluster the usual caveats regarding configuring the cluster to accept CA's and certificates used by the proxy may apply.

Reply:
It might also be helpful to mention that specifying an https:// url for a proxy that only supports http is going to result in some rather strange behavior depending on the proxy used, most proxies will report an error but the failure message may not propagate to the user/logs and look more like a network connection failure.